isis/src/base/objs/Calculator/Calculator.cpp

```diff
@@ -1001,14 +1002,18 @@ namespace Isis {
    * Print the vector at the top of the stack
    */
   void Calculator::PrintTop() {
-    if(p_valStack->empty()) return;
+    if (p_valStack->empty()) return;
 
-    std::cout << "[ ";
+    QString temp;
+    temp += "[ ";
     QVector<double> top = p_valStack->top();
-    for(int i = 0; i < (int)top.size(); i++) {
-      std::cout << top[i] << " ";
+    for (int i = 0; i < (int)top.size(); i++) {
+      temp += QString::number(top[i]);
+      temp += " ";
     }
-    std::cout << "]" << std::endl;
+    temp += "]";
+    temp.replace(QRegExp("-nan"), "nan");
```

Comment:
it might be useful to put in a comment here saying why we are replacing all "-nan" with "nan"

Reply:
Good idea!